src/lexer_test.cc

```diff
@@ -19,6 +19,16 @@
 
 using namespace std;
 
+string tok(Lexer::Token t) {
```

Comment:
nit: Ninja sources are now using std:: namespace prefixes, can you change this to use `std::string` and `std::to_string` here and below?

Reply:
sure

Reply:
sorry for double push, forgot about that the first time